tex/measure/lebesgue-int.tex

```diff
@@ -157,6 +250,18 @@ \section{Relation to Riemann integrals (or: actually computing Lebesgue integral
 	\[ \int_a^b f(x) \; dx = \int_{[a,b]} f \; d\mu. \]
 \end{theorem}
 
+Note that a Riemann integrable function is \emph{bounded},
+which means if you try to construct a function $f \colon [0, 1] \to \RR$
+in the same vein as \Cref{prob:sin_improper} by
+\[
+	f(x) = \begin{cases}
+		\frac{\sin(1/x)}{x} & x > 0 \\
+		0 & x = 0
+	\end{cases}
+\]
```

Comment:
"common mistake"? (not sure how many people would make this mistake...)

Reply:
Maybe it's clearer to write "Riemann integrable function _must be_ bounded"?